### DISCUSSION TITLE
Issue Enabling 2FA with SMS in Georgia on GitHub

### DISCUSSION THREAD

**Discussion by webview3** [community/community#86552] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

 I think it is essential to have the correct "Country or region code" available for 2FA, especially for SMS-based authentication. It seems that Georgia has been inadvertently excluded from the list of available countries or regions.

**Comment by ghostinhershell** [comment]
Hi there @webview3, 

I actually do see Georgia on the list of countries that are supported by two factor authentication: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported. Are you having difficulty using your mobile phone to authenticate?
